manifests/m/Microsoft/Bicep/0.18.4/Microsoft.Bicep.yaml

```
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.Bicep
+PackageVersion: 0.18.4
```

```suggestion
PackageVersion: 0.18.4.5664
```
This must match control panel, or AppsAndFeaturesEntries need to be added to this and all previous versions